scripts-available/CDB_CartodbfyTable.sql

```
@@ -427,8 +428,9 @@ END;
 $$ LANGUAGE 'plpgsql';
 
 
--- Find a unique relation name in the given schema, starting from the 
--- template given. If the template is already unique, just return it; 
+-- DEPRECATED: Use cartodb.CDB_Unique_Identifier since it's UTF8 Safe and length
+-- aware. Find a unique relation name in the given schema, starting from the
+-- template given. If the template is already unique, just return it;
 -- otherwise, append an increasing integer until you find a unique variant.
 CREATE OR REPLACE FUNCTION _CDB_Unique_Relation_Name(schemaname TEXT, relationname TEXT)
 RETURNS TEXT
```

I'd raise an error on deprecated functions.